costs/model-costs.json

```diff
@@ -35,10 +21,11 @@
     },
```

Comment:
This object and others below it are missing a good number of fields to be useful